src/__tests__/functional-tests/test-utils.test.tsx

```diff
@@ -7,9 +7,79 @@
 import React from 'react';
 import { render } from 'react-dom';
 import { render as renderTestingLibrary } from '@testing-library/react';
+import { pascalCase } from 'change-case';
 
+import { Modal } from '../../../lib/components';
 import Button from '../../../lib/components/button';
-import createWrapper from '../../../lib/components/test-utils/dom';
+import * as dom from '../../../lib/components/test-utils/dom';
+import * as selectors from '../../../lib/components/test-utils/selectors';
```

Comment:
What's the reason for using the wildcard export instead of the named export?

Reply:
Initially we were iterating through the `dom` and `selectors` object, but with the latest changes we don't need that anymore and we use the default export only. So removing these 👍